### ISSUE TITLE
Cannot create Panorama DeviceGroup

### ISSUE THREAD

**Issue by bvaradinov-c** [PaloAltoNetworks/pan-os-ansible#102]
### Is your feature request related to a problem?
Cannot create Panorama DeviceGroup
There is no module for DeviceGroup creation. In the same time if you pass device_group argument it says  Device group \"xxxx\" is not present."

### Describe the solution you'd like
It would be nice to have a module for DeviceGroup creation.
Example:
panos_device_group:
  name: XXXX
  parent_group: Shared
  .....

### Describe alternatives you've considered
I couldn't find alternatives. I couldn't find op command. 

### Additional context
We are trying to automation Panorama and Firewalls end to end.

**Comment by mrichardson03** [comment]
Waiting on https://github.com/PaloAltoNetworks/pan-os-python/pull/268 for hierarchy support.

**Comment by shinmog** [comment]
PR has changed to PaloAltoNetworks/pan-os-python#300

**Comment by shinmog** [comment]
New PR for device group hierarchy support:  PaloAltoNetworks/pan-os-python#321

**Comment by mrichardson03** [comment]
This is now supported by pan-os-python 1.1.0.
